frontend/src/routes/SmallBusinessSupport/SmallBusinessSupport.js

```diff
@@ -0,0 +1,41 @@
+import Box from '@mui/material/Box'
+import Container from '@mui/material/Container'
+import SectionText from 'components/SectionText'
+import sectionsData from 'mocked_api_responses/detail_data_example_small_business_support_adelaide.json'
+import sectionsLayout from 'mocked_api_responses/detail_layout_example_small_business_support.json'
+import Typography from '@mui/material/Typography'
+
+const componentMappings = {
+  text: SectionText,
+  line: null
+}
+
+const SmallBusinessSupport = () => {
+  const { sections } = sectionsLayout
+  return (
+    <Container sx={{ mt: 3 }}>
+      {sections.map((section, index) => (
+        <Box key={index} sx={{ mb: 2 }}>
+          <Typography variant="h4" gutterBottom>
+            <strong>{section.title}</strong>
+          </Typography>
```

Comment:
I think I'd prefer adding custom variants to the MUI theme when possible rather than specifying much CSS in components. For example, this could be `<Typography variant="sectionTitle">{section.title}</Typography>`. Similarly, `text`, `number`, and `simple_bar` are all meant to share text styles in the Figma design.

I'm almost definitely going to go back and forth tweaking fonts and spacing of everything a few times on this project, and having much of the project's appearance controlled by theme.js would make that easier for me.

Also, it's likely that some deployers of Atlas v2 will want to style/theme their instance to differentiate it from our main one, and I'd like to make that straightforward for them if possible.